CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ a release.
 
 ### Fixed
 - Tree: Creation of dynamic `Node::$sibling` property, which is deprecated as of PHP >= 8.2
+- Sluggable: Use `TranslationWalker` hint only for Translatable objects when looking for similar slugs (`getSimilarSlugs` method) (#100, #2530)
```

Comment:
IMO, this description conveys the idea that previous to this change, any object (even when they were not implementing `Translatable`) were being translated.
IIUC, this is not the case. Could you please confirm?

Reply:
Yes this is not the case. I meant that now we are looking for similar slugs using `TranslationWalker` but only when entities implements `Translatable` interface.